web/common.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
web/ is not going to contain a lot of data. It is going to contain hardcore code for the web interface: Angular bindings; rest api clients;